drivers/lora/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2019 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Top-level configuration file for LORA drivers.
+
+menuconfig LORA
+	bool "LoRa support"
```

Comment:
It might be a good idea to mark the LoRa support as `[EXPERIMENTAL]` here so that it is easier to tweak the API if needed.